agent/internal/config/config.go

```diff
@@ -20,10 +20,10 @@ type CommonConfiguration struct {
 	GrpcKeepalive        time.Duration `yaml:"grpcKeepalive"         env:"GRPC_KEEPALIVE"          env-default:"60s"`
 	Debug                bool          `yaml:"debug"                 env:"DEBUG"                   env-default:"false"`
 	// TODO(c3ppc3pp): custom setter to validate input
```

Comment:
The comment belongs here, custom setter being done, this is safe to be removed.

Reply:
cleaned out.

Reply:
not sure on that one: i've removed the comment, anything else which I may miss?